packages/examples/src/scenes/canvas-operations-alpha-matte.tsx

```diff
@@ -0,0 +1,105 @@
+import {makeScene2D} from '@motion-canvas/2d';
+import {AnimatedMotionCanvasIcon} from '../components/AnimatedMotionCanvasIcon';
+import {createSignal} from '@motion-canvas/core/lib/signals';
+import {linear} from '@motion-canvas/core/lib/tweening';
+import {Gradient} from '@motion-canvas/2d/lib/partials';
+import {Img, Layout, Rect, Txt} from '@motion-canvas/2d/lib/components';
+
+import valueImage from '../../assets/federico-bottos-Z3NceSeZqgI-unsplash.jpg';
```

Comment:
Could do. My rationale was to keep it local in case the file path changes or the picture is taken offline. Also means that the Docs can be used offline.